packages/components/psammead-image/README.md

```diff
@@ -72,6 +73,8 @@ Specifying the `width` and `height` allows the browser to reserve space for the
 
 The `srcset` prop is optional since some projects might not want to use the srcset attribute on images.
 
+The `fade` prop is optional and set to `false` by default. It's been used to apply a fade-in animation effect on the `Img` component.
```

Comment:
You contradict yourself here, looking at @rhell's comment?

@rhenshaw56

Reply:
@oluoluoxenfree I actually set it too `false` by default, I think it got overwritten while fixing conflicts cos @rhell and I had a chat about setting the default prop to `false`

Reply:
Yes it should be `false` by default, as we don't want it to affect front pages or indexes. We can force it on in the article.